src/avx/devices/datavideo.py

```diff
@@ -32,3 +35,86 @@ def tallyGreen(self):
 
     def tallyOff(self):
         self.sendVISCA([0x01, 0x7E, 0x01, 0x0A, 0x00, 0x03])
+
+
+class PTC150_DVIP(_PTC150Base, DVIPCamera):
+    def _tally(self, red, green):
+        self.sendVISCA([0x01, 0x7E, 0x01, 0x0A, 0x00, 0x02 * red, 0x02 * green])
+
+    def tallyRed(self):
+        self._tally(True, False)
+
+    def tallyGreen(self):
+        self._tally(False, True)
+
+    def tallyOff(self):
+        self._tally(False, False)
+
+
+class Aperture(CameraSettingEnum):
+    CLOSE = (0x00, "Closed")
+    F14 = (0x05, "F14")
+    F11 = (0x06, "F11")
+    F9_6 = (0x07, "F9.6")
+    F8 = (0x08, "F8")
+    F6_8 = (0x09, "F6.8")
+    F5_6 = (0x0A, "F5.6")
+    F4_8 = (0x0B, "F4.8")
+    F4 = (0x0C, "F4")
+    F3_4 = (0x0D, "F3.4")
+    F2_8 = (0x0E, "F2.8")
+    F2_4 = (0x0F, "F2.4")
+    F2 = (0x10, "F2")
+    F1_6 = (0x11, "F1.6")
+
+
+class Shutter(CameraSettingEnum):
+    T1 = (0x00, "1s")
```

Comment:
You can have exposures longer than the frame rate - got some very blurry results with 1s exposures while I was testing - from memory it duplicated frames rather than using a rolling buffer (so at 1080p25 you'd get 25 identical frames, followed by another set of 25, rather than each frame being made of the sum of the previous 24). That might not be entirely accurate - I wasn't really paying much attention and it was a few weeks ago now!